postconf: add page (#10784)
tldr-pages/tldr: pages/linux/postconf.md

# postconf

> > Postfix coniguration utility.
> > This command displays the values of main.cf configuration parameters by default and warns about possible mis-typed parameter names. It can also change the main.cf configuration parameter values.

Reviewer: ```suggestion
> Postfix configuration utility.
> This command displays the values of the `main.cf` configuration parameters by default and warns about possible mistyped parameter names. It can also change the `main.cf` configuration parameter values.
```

Author: Done.

> More information: <https://manned.org/postconf>.

> - Specify the directory of the main.cf configuration file instead of the default configuration directory:

Reviewer: ```suggestion
- Specify the directory of the `main.cf` configuration file instead of the default configuration directory:
```

`postconf -c {{path/to/configuration_directory}}`

> - Edit the `main.cf` configuration file and update parameter settings with the "name=value" pairs on the command line:

Reviewer: ```suggestion
- Edit the `main.cf` configuration file and update parameter settings with the "name=value" pairs:
```

Author: Done

> - Specify the directory of the main.cf configuration file instead of the default configuration directory:
> 
> `postconf -c {{config_dir}}`
> 
> - Edit the main.cf configuration file and update parameter settings with the "name=value" pairs on command line:
> 
> `postconf -e`
> 
> - Print the default parameter settings of main.cf instead of actual settings:
> 
> `postconf -d`

Reviewer: ```suggestion
- Specify the directory of the main.cf configuration file instead of the default configuration directory:

`postconf -c {{path/to/configuration_directory}}`

- Edit the `main.cf` configuration file and update parameter settings with the "name=value" pairs on the command line:

`postconf -e`

- Print the default parameter settings of the `main.cf` instead of the actual settings:

`postconf -d`
```

We generally use the `{{path/to/placeholder}}` format for the path placeholders. And regarding the file name, we can highlight it with backticks.

Author: Done.

- Display parameters only from the specified class. The class can be one of builtin, service, user or all:

`postconf -C {{class}}`

> - List available SASL plug-in tyes for the Postfix SMTP server. The plug-in type is selected with the smtpd_sasl_type configuration parameter by specifying cyrus or dovecot as the name:
> 
> `postconf -a`
> 
> - List the names of all supported lookup table types. Lookup tables are specified as type:name in configuration files where type can be btree, cdb, hash, mysql etc:
> 
> `postconf -m`

Reviewer: ```suggestion
- List available SASL plug-in types for the Postfix SMTP server. The plug-in type is selected with the `smtpd_sasl_type` configuration parameter by specifying `cyrus` or `dovecot` as the name:

`postconf -a`

- List the names of all supported lookup table types. Lookup tables are specified as `type:name` in configuration files where the type can be `btree`, `cdb`, `hash`, `mysql`, etc:

`postconf -m`
```

Author: Done
